src/Model/GServer.php

```diff
@@ -562,6 +653,11 @@ public static function detect(string $url, string $network = '', bool $only_node
 			$serverdata = self::detectNetworkViaContacts($url, $serverdata);
 		}
 
+		if (($serverdata['network'] == Protocol::PHANTOM) && in_array($serverdata['detection-method'], [self::DETECT_MANUAL, self::DETECT_BODY])) {
+			self::setFailure($url);
```

Comment:
Please rename the method:
```suggestion
			self::setFailureByUrl($url);
```

Reply:
Done